tools/ccm-service-kit/composer/readme.md

```diff
@@ -0,0 +1,138 @@
+# Multi Cloud Billing Solution Cloud Composer set up
+
+## Introduction
+
+We recommend Composer to orchestrate the whole pipeline. You will find the scripts in the git repository. It is important to understand that this architecture is just a standard and each customer is free to choose another tool to develop this ETL.
+
+## Airflow scripts
+Under the composer directory you will find the following scripts:
+
+- dags
+    - This folder will contains the dags to be executed in the Composer server and the dependencies.
+    - This folder should be sync with the composer bucket in your CI/CD pipelines to keep the code updated in the airflow server.
+- dags/dependencies
+    - This directory contains the following folders:
+        - operators: we created the LoadFileFromAPI operator to get a file over an API request and load it into GCS.You can extend this operator based on your needs.
+        - queries:this folder has all the queries needed to create and refresh the unified schema. If you need, you can modify or extend the queries.
+        - scripts: under this folder you will find a script that executes a transfer service job and the variable_reader script.
+        - variables: you will find a json file with the variables that the dags will be using. You have to edit this file.
+
+## How to use it
+
+First you need to edit the following variables:
+
+1. Go to **composer/dags/dependencies/variables/variables.json**
+
+```
+{
+    "project_id" : "ccm-dev-sandbox",
+    "ccm_bucket" : "ccm-zgnzp0",
+    "prefix_azure_lz" : "azure-lz",
+    "prefix_aws_lz" : "aws-lz",
+    "prefix_hist_azure" : "azure-historical",
+    "prefix_hist_aws" : "aws-historical",
+    "sts_id_azure" : "transferJobs/13017487796363048406",
+    "sts_id_aws" : "transferJobs/16024774301148900524",
+    "gcp_billing_table" : "gcp_billing",
+    "ccm_staging_dataset" : "cloud_cost_staging",
+    "ccm_final_dataset" : "cloud_cost_final"
+}
```

Comment:
Please restrain from using the actual name of the resources and use place holders. For example instead of `"ccm-dev-sandbox"` use `<project-name>`